pkg/importer/dataStream_test.go

```diff
@@ -162,31 +157,36 @@ func startHTTPServer(port int, dir string) (*http.Server, error) {
 	return server, nil
 }
 
+func createTestServer(imageDir string) *httptest.Server {
+	return httptest.NewServer(http.FileServer(http.Dir(imageDir)))
+}
+
 func TestNewDataStream(t *testing.T) {
 	type args struct {
 		endpt  string
 		accKey string
 		secKey string
 	}
-
 	imageDir, _ := filepath.Abs(TestImagesDir)
```

Comment:
why is imageDir defined here vs. as a global var referenced by all tests?

Reply:
Most tests already had it defined there, I didn't change it, but good suggestion, I will make that change.

Reply:
The latest commit (6th) to 443 has this in place